Lib/statistics.py

```diff
@@ -932,18 +932,18 @@ def correlation(x, y, /):
         raise StatisticsError('at least one of the inputs is constant')
 
 
-LinearRegression = namedtuple('LinearRegression', ['intercept', 'slope'])
+LinearRegression = namedtuple('LinearRegression', ['slope', 'intercept'])
```

Comment:
Minor nit, make this a tuple instead of a list:  ``('slope', 'intercept')``.

Reply:
Done.